src/Plugins/BotSharp.Plugin.Twilio/Services/TwilioMessageQueueService.cs

```diff
@@ -122,9 +147,7 @@ private async Task ProcessUserMessageAsync(CallerMessage message)
             }
             // add frequency short words
             hints.AddRange(["yes", "no", "correct", "right"]);
-            reply.Hints = string.Join(", ", hints.Select(x => x.ToLower()).Distinct().Reverse());
```

Comment:
Have to keep `reply.Hints`

Reply:
Extracted the method and moved the location